app.base.json

```diff
@@ -20,7 +20,8 @@
     "showThorButton": false,
     "showVulaanControls": false,
     "useProjectenSignalType": false,
-    "enableForwardIncidentToExternal": false
+    "enableForwardIncidentToExternal": false,
+    "showRealTimeDashboard": true
```

Comment:
realtime is 1 word

Reply:
Right, changed it to just dashboard since Jelle mentioned there will be two dashboards.